rust/operator-binary/src/superset_controller.rs

```diff
@@ -289,25 +388,29 @@ fn build_server_rolegroup_statefulset(
     let statsd_exporter_image =
         format!("docker.stackable.tech/prom/statsd-exporter:{statsd_exporter_version}");
 
-    let env = node_config
+    let mut cb = ContainerBuilder::new("superset");
+
+    for (name, value) in node_config
         .get(&PropertyNameKind::Env)
-        .and_then(|vars| vars.get(SupersetConfig::CREDENTIALS_SECRET_PROPERTY))
-        .map(|secret| {
-            vec![
-                env_var_from_secret("SECRET_KEY", secret, "connections.secretKey"),
-                env_var_from_secret(
-                    "SQLALCHEMY_DATABASE_URI",
-                    secret,
-                    "connections.sqlalchemyDatabaseUri",
-                ),
-            ]
-        })
-        .unwrap_or_default();
+        .cloned()
+        .unwrap_or_default()
+    {
+        if name == SupersetConfig::CREDENTIALS_SECRET_PROPERTY {
```

Comment:
Nit: Blacklisting secrets seems dangerous, so if you have an idea how to **easily** convert this to a whitelist or so would be great

Reply:
The environment variable from the product config is just expanded into two environment variables in this case. The content of the secret is not exposed outside the container. If an environment variable with a secret name "slips through" then only the name of the secret is visible but not its content. The name of the secret is nevertheless visible in the custom resource. So it is no secret.

Reply:
Alright, i got little on the wrong track. Thanks!